internal/bminventory/inventory.go

```diff
@@ -555,6 +555,8 @@ func (b *bareMetalInventory) RegisterCluster(ctx context.Context, params install
 		cluster.SSHPublicKey = sshPublicKey
 	}
 
+	//HERE
```

Comment:
?

Reply:
Will be used here, not implemented yet.

It won't work until https://issues.redhat.com/browse/SDB-1656 is implemented and that our service-account in ocm-resources can list/post subscription.

This is why I pushed only an early version of the PR to get some first review from @rollandf and @danielerez.

Will be added soon.

Reply:
Added now.